### ISSUE TITLE
Fetal tachycardia

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9603]
**Preferred term label:** Fetal tachycardia

**Synonyms**

**Definition (free text, please give PubMed ID)**

"Among 33 patients with identified MAT, 19 were boys and 27 (82%) were younger than 1 
year old at the time of diagnosis (infant group). The median age at diagnosis of MAT was 1.7 
months (range, 0 days–14.2 years). Ten patients (30%) had fetal arrhythmias/tachycardia on 
fetal monitoring, while nine were first detected incidentally on physical examination (27%). "
PMID  29441747

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA 3282 Multifocal atrial tachycardia (MAT)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@carolina-f 
sorry for the delay

Adding new term: 
- ID: HP:6000264
- label: Fetal tachycardia
- parents: [Abnormal fetal cardiovascular physiology (HP:0034190)](https://hpo.jax.org/app/browse/term/HP:0034190)
- pmid: PMID:30244488
- def: The number of beats per minute (bpm) of the fetal heart is above the upper limit of the normal for the gestational age. Fetal tachycardia is thus defined as a heart rate greater than 160-180 bpm.
- comment: In the first trimester of pregnancy, the fetal heart rate (FHR)increases from 125 beats per minute (bpm) at 6 weeks to about175 bpm at 9 weeks, followed by a gradual decrease to about160 bpm at 14 weeks, noted in fetuses conceived by assisted reproductive procedures.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9603
- synonyms:
